crates/pixi_command_dispatcher/src/instantiate_tool_env/mod.rs

```diff
@@ -152,61 +161,31 @@ impl InstantiateToolEnvironmentSpec {
     pub async fn instantiate(
         self,
         command_queue: CommandDispatcher,
-    ) -> Result<Prefix, CommandDispatcherError<InstantiateToolEnvironmentError>> {
+    ) -> Result<
+        InstantiateToolEnvironmentResult,
+        CommandDispatcherError<InstantiateToolEnvironmentError>,
+    > {
         tracing::debug!(
             "Installing tool env for: {}",
             &self.requirement.0.as_source()
         );
 
         // Determine the cache key for the environment.
         let cache_key = self.cache_key();
-        let name = self.requirement.0.as_source().to_string();
```

Comment:
I changed the code below because I always want to solve the environment to ensure we get the latest versions.

Reply:
😿